CHANGELOG.rst

```diff
@@ -2,6 +2,239 @@
+  * GPU support for build files (`#624 <https://github.com/ros-infrastructure/ros_buildfarm/pull/624>`_)
+  * Implement the abichecker run on devel builds (`#681 <https://github.com/ros-infrastructure/ros_buildfarm/pull/681>`_)
+  * Add scripts to build blocked source entries page (`#674 <https://github.com/ros-infrastructure/ros_buildfarm/pull/674>`_)
+
+* Improvements
+
+  * Convert to subprocess model for flake8 test (`#1075 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1075>`_)
+  * Make apt work with unsigned repositories (`#1070 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1070>`_)
+  * Add flags to separate generate_release_script parts (`#1066 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1066>`_)
+  * Delete any existing UID user in Dockerfile (`#1061 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1061>`_)
+  * Support unsigned apt repositories (`#947 <https://github.com/ros-infrastructure/ros_buildfarm/pull/947>`_)
+  * Switch to rosdistro.get_package_condition_context API (`#1065 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1065>`_)
+  * Add logic for excluding group workaround dependencies (`#1040 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1040>`_)
+  * Add new seccomp warning text to ignore pattern. (`#1050 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1050>`_)
+  * Use tar and ssh to publish package metadata files. (`#1049 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1049>`_)
+  * Clean up any dangling cidfiles (Container ID files) in workspace. (`#1047 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1047>`_)
+  * Avoid jenkinsapi trying to fetch all jobs. (`#1045 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1045>`_)
+  * Make sure to call generate_doc_maintenance_jobs.py for rosdoc2. (`#1043 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1043>`_)
+  * Set PODMAN_USERNS=keep-id when invoking 'docker run' (`#1032 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1032>`_) (`#1069 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1069>`_)
+  * Use environment to configure breaking system packages. (`#1036 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1036>`_)
+  * Handle RPM *-SPECPARTS build subdirectory (`#1033 <https://github.com/ros-infrastructure/ros_buildfarm/pull/1033>`_)
```

Comment:
Should the `*` there be escaped? GH's rST renderer starts rendering all text as italics after it.

Reply:
Ah thanks, good call. This is why we do reviews :)

Reply:
Fixed in latest push. Same change was applied to line 78.